app/components/navigation_bar/view.html.erb

```diff
@@ -0,0 +1,27 @@
+<% if user_signed_in? %>
+  <div class="moj-primary-navigation">
+    <div class="moj-primary-navigation__container">
+      <div class="moj-primary-navigation__nav">
+        <nav class="moj-primary-navigation" aria-label="Primary navigation">
+          <ul class="moj-primary-navigation__list">
+            <li class="moj-primary-navigation__item">
+              <%= govuk_link_to "Home", home_path, { class: "moj-primary-navigation__link" } %>
+            </li>
+            <li class ="moj-primary-navigation__item">
+              <%= govuk_link_to "Trainee records", trainees_path, { class: "moj-primary-navigation__link" } %>
+            </li>
+          </ul>
+        </nav>
+      </div>
+      <div class="moj-primary-navigation__search">
+        <nav class="moj-primary-navigation" aria-label="Primary navigation">
+          <ul class="moj-primary-navigation__list">
+            <li class ="moj-primary-navigation__item">
+              <%= govuk_link_to "Bulk actions", href='#', { class: "moj-primary-navigation__link" } %>
+            </li>
+          </ul>
+        </nav>
+      </div>
```

Comment:
Bulk actions haven't been built yet, so I'd kill this.

Reply:
Additionally - when we come to building this I think we'd want it in the same nav, not in an extra one.

Reply:
cool, removed this